CONTRIBUTING.md

```diff
@@ -0,0 +1,103 @@
+# Contributing to uqcsbot
+
+:+1::tada: First off, thanks for taking the time to contribute! :tada::+1:
+
+The following is a set of guidelines for contributing to uqcsbot. These are mostly guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the [UQCS Code of Conduct](https://github.com/UQComputingSociety/code-of-conduct/blob/master/README.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [contact@uqcs.org.au](mailto:contact@uqcs.org.au).
+
+## Got a Question or Problem?
+
+If you would like to chat about the question in real-time, you can reach out via [our slack channel](https://uqcs.slack.com/).
```

Comment:
Perhaps point explicitly to #projects as somewhere to ask uqcsbot-specific questions? (and/or mention #python)

Reply:
good idea :^) Done.